webp/view/src/main/cpp/webp_view.cpp

```diff
@@ -189,6 +189,7 @@ bool Engine::PrepareDrawing(void) {
             break;
         case WINDOW_FORMAT_RGBA_8888:
             descriptor.format_ = SurfaceFormat::SURFACE_FORMAT_RGBA_8888;
+            [[fallthrough]];
```

Comment:
add a `break;` instead: should be supportable; otherwise
change the build to request for 17 [here]()https://github.com/android/ndk-samples/blob/master/webp/view/src/main/cpp/CMakeLists.txt#L47?